gizmos/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE HTML>
+<html>
+<head>
```

Comment:
You might want to indent everything one tab inside of the `<html>` element so the structure looks like this:

```html
<!DOCTYPE html>
<html>
  <head>
    ...
  </head>
  
  <body>
    ...
  </body>
</html>
```